crates/sui/src/upgrade_compatibility.rs

```diff
@@ -813,6 +821,43 @@ fn module_compatibility_error_diag(
     Ok(diags)
 }
 
+fn missing_module_diag(
+    module_name: &Identifier,
+    package_path: &PathBuf,
+) -> Result<Diagnostics, Error> {
+    const PACKAGE_TABLE: &str = "[package]";
```

Comment:
@tzakian is this string stored somewhere already in the toml parsing?

Reply:
doesnt appear to be in the file_map, I'll just read it for now we might have this later with some work from @tzakian